src/utils/storage.js

```diff
@@ -73,16 +81,24 @@ export const addWordToUserList = async (word) => {
   try {
     const user = await getUser();
     let userData = await getUserData(user);
-    if (userData === DEFAULT_USER_ERROR) return false;
-    if (!userData) userData = { wordList: [] };
-    userData.wordList.push({
-      word: getLemma(word),
+
+    if (userData === DEFAULT_USER_ERROR) {
+      return false;
+    }
+
+    if (!userData) {
+      userData = {
+        wordList: getBlankUserWordList(),
+      };
+    }
+    userData.wordList.one[getTargetLemma(word)] = {
       otherInfo: { date: new Date() },
-    });
+    };
     addToWordList(word);
     await AsyncStorage.setItem(`@${user}`, JSON.stringify(userData));
     return true;
   } catch (error) {
+    console.log(error);
```

Comment:
It's a minor thing, but if you wanted these log statements to garner more attention you could use `console.error(error)` instead.